src/insns/load_tag_perms.adoc

```diff
@@ -0,0 +1,11 @@
+Resulting value of `cd`::
+The tag value written to `cd` is 0 if the tag of the memory location loaded is
+0 or the authorizing capability (<<ddc>> or `cs1`) does not grant <<c_perm>>.
++
+If the authorizing capability has both <<em_perm>> and <<w_perm>> unset, and the tag of `cd` is 1 and `cd` is not sealed, then an implicit <<ACPERM>> clearing <<w_perm>> and <<em_perm>> is performed to obtain the final permissions on `cd`.
+
+NOTE: Missing <<em_perm>> does not affect untagged values since this could result in surprising bit patterns when copying non-capability data.
+Similarly, sealed capabilities are not modified as they are not directly dereferenceable.
+
+NOTE: The implicit <<ACPERM>> does not introduce a new dependency on the loaded data since the actual masking of permissions can be deferred until the loaded capability is dereferenced or the metadata bits are inspected using <<GCPERM>> or <<GCHI>>.
```

Comment:
I think it does introduce a dependency, and that might well be what some microarchitectures do for simplicity. Its just that some other microarchitectures can work around this dependency using the hinted trick here.

Reply:
Reworded, hopefully better now?